gammapy/cube/cube_pipe.py

```diff
@@ -0,0 +1,270 @@
+
+        Parameters
+        ----------
+        radius : float
+            Disk radius in pixels.
+        """
+        disk = Tophat2DKernel(radius)
+        disk.normalize('peak')
+        list_kernel = [disk.array] * (len(self.significance_cube.energies()))
+        counts = self.counts_cube.convolve(list_kernel)
+        bkg = self.bkg_cube.convolve(list_kernel)
+        self.significance_cube.data = significance(counts.data, bkg.data)
+
+    def make_excess_cube(self):
+        """Compute excess cube between counts and bkg cubes."""
+        self.excess_cube.data = self.counts_cube - self.bkg_cube
+
+
+class StackedObsCubeMaker(object):
+    """Compute stacked cubes for many observations.
+
+    The computed cubes are stored in a `~gammapy.cube.SkyCube` with the following name:
+
+    * ``counts_cube`` : Counts
+    * ``bkg_cube`` : Background model
+    * ``exposure_cube`` : Exposure
+    * ``excess_cube`` : Excess
+    * ``significance_cube`` : Significance
+
+    Parameters
+    ----------
+    empty_cube_images : `~gammapy.cube.SkyCube`
+        Reference Cube for images in reco energy
+    empty_exposure_cube : `~gammapy.cube.SkyCube`
+        Reference Cube for exposure in true energy
+    offset_band : `astropy.coordinates.Angle`
+        Offset band selection of the events to fill the cubes
+    data_store : `~gammapy.data.DataStore`
+        Data store
+    obs_table : `~astropy.table.Table`
+        Required columns: OBS_ID
+    exclusion_mask : `~gammapy.cube.SkyCube`
+        Cube of `~gammapy.image.SkyMask`
+    save_bkg_scale: bool
+        True if you want to save the normalisation of the bkg for each run in a `Table` table_bkg_norm with two columns:
+         "OBS_ID" and "bkg_scale"
+    """
+
+    def __init__(self, empty_cube_images, empty_exposure_cube=None, offset_band=None, data_store=None, obs_table=None,
+                 exclusion_mask=None, save_bkg_scale=True):
+
+        self.empty_cube_images = empty_cube_images
+        if not empty_exposure_cube:
+            self.empty_exposure_cube = SkyCube.empty_like(empty_cube_images)
+        else:
+            self.empty_exposure_cube = empty_exposure_cube
+
+        self.counts_cube = SkyCube.empty_like(empty_cube_images)
+        self.bkg_cube = SkyCube.empty_like(empty_cube_images)
+        self.significance_cube = SkyCube.empty_like(empty_cube_images)
+        self.excess_cube = SkyCube.empty_like(empty_cube_images)
+        self.exposure_cube = SkyCube.empty_like(empty_exposure_cube)
+
+        self.data_store = data_store
+        self.obs_table = obs_table
+        self.offset_band = offset_band
+
+        self.header = empty_cube_images.sky_image_ref.to_image_hdu().header
+        self.cube_exclusion_mask = exclusion_mask
+
+        self.save_bkg_scale = save_bkg_scale
+        if self.save_bkg_scale:
+            self.table_bkg_scale = Table(names=["OBS_ID", "bkg_scale"])
+
+    def make_images(self, make_background_image=False, bkg_norm=True, radius=10):
```

Comment:
This method is kind of a misnomer, because it actually computes cubes not images...maybe rename to `make_cubes()`?